src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,13 +1,15 @@
-<div id="sfwdt{{ token }}" class="sf-toolbar" style="display: none"></div>
```

Comment:
You're right, it was only necessary when I injected the scripts before `<title>`. That being said does this matter?